shoop_tests/core/test_basic_order.py

```diff
@@ -85,7 +86,7 @@ def create_order(request, creator, customer, product):
 
     shipment = order.create_shipment_of_all_products(supplier=supplier)
     assert shipment.total_products == 5, "All products were shipped"
-    assert shipment.weight == product.net_weight * 5, "Gravity works"
+    assert shipment.weight == product.net_weight * 5 / decimal.Decimal("1000"), "Gravity works"
```

Comment:
using decimal.Decimal("1000") makes this harder to read. you can use just 1000, since the left side is already Decimal.